Tests/Unit/MSFT_xFTP.test.ps1

```diff
@@ -0,0 +1,2257 @@
+                    -ClientOnly
+            
+            $MockFtpServerInfo = @(
+                    @{
+                        userIsolation = @{
+                            mode = 'IsolateAllDirectories'
+                                         }
+                     }
+
+                    @{
+                        directoryBrowse = @{
+                            showFlags = 'LongDate'
+                                           }
+                     }
+
+                    @{
+                        security = @{
+                            ssl =@(New-Object -TypeName PSObject -Property @{
+                                serverCertHash       = 'EF8D5381178A622886A30CBBB46BBA8F4AFAAC97'
+                                serverCertStoreName  = 'MY'
+                                ssl128               = 'True'
+                                controlChannelPolicy = 'SslAllow'
+                                dataChannelPolicy    = 'SslAllow'
+                            })
+                        }
+                   }
+            )
+
+            $MockParameters = @{
+                Ensure               = 'Present'
+                Name                 = 'MockFtp'
+                PhysicalPath         = 'C:\NonExistent'
+                State                = 'Started'
+                ApplicationPool      = 'MockFtpPool'
+                AuthorizationInfo    = $MockAuthorizationInfo
+                BindingInfo          = $MockBindingInfo
+                SslInfo              = $MockSslInfo
+                LogPath              = '%SystemDrive%\LogFiles'
+                LogFlags             = @('Date','Time','ClientIP','UserName','ServerIP','Method')
+                LogPeriod            = 'Daily'
+                LoglocalTimeRollover = $false
+                DirectoryBrowseFlags = 'StyleUnix'
+                UserIsolation        = 'StartInUsersDirectory'
+            }
+
+            $MockWebsite = @{
+                Name                 = 'MockFtp'
+                PhysicalPath         = 'C:\NonExistent'
+                State                = 'Started'
+                ApplicationPool      = 'MockFtpPool'
+                AuthenticationInfo   = $AuthenticationInfo
+                AuthorizationInfo    = $AuthorizationInfo
+                Bindings             = @{Collection = @($MockBindingInfo)}
+                logfile              = $MockLogOutput
+                ftpServer            = $MockFtpServerInfo
+                Count                = 1
+            }
+
+            Context 'Website does not exist' {
+                Mock -CommandName Get-Website
+
+                $Result = Test-TargetResource `
+                            -Ensure $MockParameters.Ensure `
+                            -Name $MockParameters.Name
+
+                It 'should return False' {
+                    $Result | Should Be $false
+                }
+            }
+
+            Context 'Check PhysicalPath is different' {
```

Comment:
I am not sure what the word 'different' means wrt these tests. i would change it to something more concrete. 
Same comment for the other context descriptions following this one